packages/templating/plugin/html_scanner.js

```diff
@@ -169,8 +171,15 @@ html_scanner = {
           templateDotNameLiteral + ", " + renderFuncCode + ");\n";
       } else {
         // <body>
-        if (hasAttribs)
-          throwParseError("Attributes on <body> not supported");
+        if (hasAttribs) {
+          for(var attr in attribs) {
```

Comment:
It should already be imported, I think all Meteor files have it no matter what. Try just using it?